v8env/src/document.ts

```diff
@@ -152,6 +159,13 @@ class DocumentParser {
 		}
 		this.parser.end()
 	}
+
+	parseSync(arg: any) {
```

Comment:
This method is being called [a few lines above](https://github.com/superfly/fly/pull/126/files#diff-1df6650cae8f8cefe6dd5d506b48b513R150) but didn't exist. I can only assume that there's either a method being added somewhere else or it's a codepath not getting hit. Depending on the context I'll fix or remove this.

Reply:
Definitely was just never getting used. We haven't used any of the document streaming stuff yet,